code/style.css

```diff
@@ -1,4 +1,380 @@
+}
+
+/* ---------- DAILY-POLAROIDS ------------- */
+
+.daily-polaroids {
+  background-color: rgb(240, 236, 236);
+}
+
+.daily-polaroid-container {
+  display: flex;
+  justify-content: center;
+  flex-flow: row wrap;
+}
+
+.daily-polaroid-header {
+  text-align: center;
+  padding-top: 20px;
+  padding-bottom: 40px;
+}
+
+.daily-polaroid {
+  height: 300px;
+  width: 250px;
+  object-fit: cover;
+  display: block;
+}
+
+.polaroid {
+  background: #ffffff;
+  display: inline-block;
+  margin-bottom: 20px;
+  padding: 15px 15px 30px;
+  text-align: center;
+  text-decoration: none;
+  -webkit-box-shadow: 0 4px 6px rgba(0, 0, 0, 0.3);
+  -moz-box-shadow: 0 4px 6px rgba(0, 0, 0, 0.3);
+  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.3);
+  -webkit-transition: all 0.2s linear;
+  -moz-transition: all 0.2s linear;
+  transition: all 0.2s linear;
+}
+
+.polaroid-text {
+  padding-top: 20px;
+}
+
+.flip-card-back {
+  display: none;
+}
+
+/* ---------- IPAD ADAPTATIONS------------- */
+
+@media only screen and (min-width: 668px) and (max-width: 1024px) {
+  .header {
+    justify-content: space-evenly;
+  }
+
+  .header :nth-child(1) {
+    order: 2;
+  }
+  .header :nth-child(2) {
+    order: 0;
+  }
+  .header :nth-child(3) {
+    order: 1;
+  }
+  .header :nth-child(4) {
+    order: 3;
+  }
+  .header :nth-child(5) {
+    order: 4;
+  }
+
+  .icon-and-text {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    margin-right: 20px;
+  }
+
+  .big-news-container {
+    flex-direction: row;
+    margin-bottom: 30px;
+  }
+
+  .big-news-image {
+    flex: 1;
+    height: 385px;
+    object-fit: cover;
+  }
+
+  .big-news-text-container {
+    flex: 1;
+    padding: 0px 20px 0px 20px;
+  }
+
+  .other-news-container {
+    flex-flow: row wrap;
+    justify-content: space-evenly;
+  }
+
+  .other-news {
+    flex-direction: column;
+    width: auto;
+    max-width: none;
+  }
+
+  .other-news-pictures {
+    max-width: 100%;
+    max-height: 200px;
+    margin-top: 10px;
+  }
+  .other-news-text-container {
+    max-width: 300px;
+    margin-top: 10px;
+    height: auto;
+  }
+
+  .item-2,
+  .item-4,
+  .item-6 {
+    flex-direction: column-reverse;
+  }
+
+  .daily-polaroid-container {
+    justify-content: space-evenly;
+  }
+
+  .daily-polaroid {
+    height: 300px;
+    width: 250px;
+    object-fit: cover;
+    display: block;
+  }
+}
+
+/* ---------- DESKTOP ADAPTATIONS------------- */
+
+@media only screen and (min-width: 1025px) {
+  body {
+    background-color: rgb(236, 230, 226);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .header {
+    justify-content: flex-end;
+    max-width: 1700px;
+    min-width: 1000px;
+  }
+
+  .a-logotype {
+    margin-right: auto;
+  }
+
+  .icon-and-text {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    margin-right: 20px;
+  }
+
+  .icon-and-text:hover {
+    font-size: 18px;
+    transition: 1s ease-out;
```

Comment:
Is this the animation? Im impressed!

Reply:
Yeah it is a part of the animation =) thanks!!